config/locales/en.yml

```diff
@@ -112,3 +112,11 @@ en:
     thank_you_6: One of the greatest gifts you can give is your time!
     thank_you_7: Those who can do, do. Those who can do more, volunteer.
     thank_you_8: Volunteers do not necessarily have the time, they just have the heart.
+  errors:
```

Comment:
@DrewAPeterson7671 maybe this is where all the test issues are coming from?

Reply:
It absolutely was!  Only one code climate issue to deal with.  Everything else passed.